### PR TITLE
All Sites: Make meta description site specific

### LINKED ISSUE #261: Make meta description site specific
Currently all sites share this:

`<meta name="description" content="jQuery: The Write Less, Do More, JavaScript Library">`

We should have a description for each site, e.g. "QUnit: A JavaScript Unit Testing framework." for the QUnit sites, or "jQuery UI is a curated set of user interface interactions, effects, widgets, and themes built on top of the jQuery JavaScript Library. Whether you're building highly interactive web applications or you just need to add a date picker to a form control, jQuery UI is the perfect choice." for the jQuery UI sites.

### PR DESCRIPTION
**Pull request by sigorilla**
Hi everyone!

Could you write a description for all sites so that I could complete this PR? Current descriptions I got from the issue #261.

Fixes #261

### CODE REVIEW

File: themes/jquery/functions.jquery.php [hunk 1]
```diff
@@ -177,3 +177,20 @@ function jq_logo_link() {
...
 }
+
+/**
+ * Meta description about current site
+ * Setup descriptions in 'sites.php'
+ *
+ * @return [String] description
+ */
+function jq_meta_description() {
+	$default = 'jQuery: The Write Less, Do More, JavaScript Library';
```
Comment (scottgonzalez) [inline themes/jquery/functions.jquery.php]: I'd actually prefer if we default to just not having a description. Any default will be wrong for the majority of sites.

Comment (sigorilla) [inline themes/jquery/functions.jquery.php]: OK, I set empty string for default.

**Comment by sigorilla** [comment]
ping

**Comment by mgol** [comment]
cc @agcolom

**Pull request closed without merging**